flow/mutators_stack_unittests.cc

```diff
@@ -85,10 +85,19 @@ TEST(MutatorsStack, PushOpacity) {
   int alpha = 240;
   stack.PushOpacity(alpha);
   auto iter = stack.Bottom();
-  ASSERT_TRUE(iter->get()->GetType() == MutatorType::opacity);
+  ASSERT_TRUE(iter->get()->GetType() == MutatorType::kOpacity);
   ASSERT_TRUE(iter->get()->GetAlpha() == 240);
 }
 
+TEST(MutatorsStack, PushBackdropFilter) {
```

Comment:
It couldn't hurt to add a test to confirm that the == method on the mutators works for all types, including if they are fed copies of the original data (i.e. they compare by content rather than reference).